scripts/eia/doi_Chinyanja-Solidaridad-Soy-NOT.R

```diff
@@ -60,10 +62,11 @@ carob_script <- function(path) {
       variety= tolower(r$variety),
       #crop_system= r$intercrop.sole,
       planting_time= r$Time.of.planting,
```

Comment:
I don't think _planting_time_ is useful since we have the `planting_date`... But I guess it could be as a class.